cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorTest.java

```diff
@@ -5,37 +5,46 @@
 package com.linkedin.kafka.cruisecontrol.detector;
 
 import com.linkedin.cruisecontrol.detector.Anomaly;
+import com.linkedin.cruisecontrol.detector.AnomalyType;
```

Comment:
Can we add a test case to verify that Anomaly Detector takes the priorities into account -- i.e. `PriorityBlockingQueue` for anomalies.